UnityProject/Assets/Scripts/Core/Physics/UniversalObjectPhysics.RightClick.cs

```diff
@@ -39,16 +39,6 @@ public virtual RightClickableResult GenerateRightClickOptions()
 				}
 			}
 
-			if (canResetRotationFromRightClick && rotationTarget != null)
```

Comment:
This was something that was requested to be added a while ago. Why are you removing this?

Reply:
oh, Thought it was for players only?

Reply:
no, it's for items
`canResetRotationFromRightClick` is enabled only for items